feature/system/tests/system_base_test/hostname_test.go

```diff
@@ -57,13 +58,16 @@ func TestHostname(t *testing.T) {
 				}
 			})
 
-			t.Run("Get Hostname Telemetry", func(t *testing.T) {
-				stateGot := gnmi.Await(t, dut, state.State(), 1*time.Minute, testCase.hostname)
-				if got, _ := stateGot.Val(); got != testCase.hostname {
-					t.Errorf("Telemetry hostname: got %v, want %s", stateGot, testCase.hostname)
-				}
-			})
-
+			if deviations.CliTakesPrecedenceOverOC(ondatra.DUT(t, "dut")) {
```

Comment:
You don't need to call ondatra.DUT(t, "dut") here -- you already have a reference to the DUT. This should just be:
`if deviations.CliTakesPrecedenceOverOC(dut)`

Reply:
Done